web/views/note_view.ex

```diff
@@ -1,4 +1,8 @@
 defmodule Notox.NoteView do
   use Notox.Web, :view
 
+  def render_markdown(markdown) do
+    import Earmark
+    Earmark.to_html(markdown) |> raw
```

Comment:
The plan right now is to add something like `escape_html` to the `Note` changeset right now. Sounds pretty easy, but I'm having a tough time figuring out how to destructure a changeset to do this.